level_2/1024.py

```diff
@@ -44,7 +44,7 @@ def cast_vote(url, cookie, payload):
 
 get_initial_data(endpoint, hodor_cookie, hodor_payload)
```

Comment:
Better to create a Class and store attributes instead of storing global variables

Reply:
This is actually what I wanted to ask. What kind of class should I create to encapsulate all of this data and behavior?